src/localization/localization.ts

```diff
@@ -49,13 +48,30 @@ const localization = {
             propStorybookStoriesInfo: "Names of stories defined under `.stories.js` of a component."
         },
         config: {
+            custom: {
```

Comment:
@berkcebi, @borulday Please check localizations, these are used in
[src/coco/config/fileChange/configRenameUtil.ts](https://github.com/zeplin/vscode-extension/blob/87997cbb9a5c893dec67e74da57a69083ecdc137/src/coco/config/fileChange/configRenameUtil.ts)
[src/coco/config/flow/configFlow.ts](https://github.com/zeplin/vscode-extension/blob/87997cbb9a5c893dec67e74da57a69083ecdc137/src/coco/config/flow/configFlow.ts)